web/tests/buddy_data.test.cjs

```diff
@@ -424,7 +425,7 @@ test("get_conversation_participants", ({override_rewire}) => {
 
     const rome_sub = {name: "Rome", subscribed: true, stream_id: 1001};
     stream_data.add_sub(rome_sub);
-    peer_data.set_subscribers(rome_sub.stream_id, [selma.user_id]);
+    peer_data.set_subscribers(rome_sub.stream_id, [selma.user_id, me.user_id]);
```

Comment:
I'm curious if this was needed to stop a test from failing after you made these changes (and if so, why)! Or if it just made more sense to have `me` as a subscriber because they're subscribed to this channel.

Reply:
No, this change was not to stop a test from failing. It was just me noticing that the `rome_sub` object had them as `subscribed: true`. And if that's true, then we'd expect the peer_data to have the current user as a subscriber. I can break this into a separate commit to make that clearer.

Reply:
Yeah I think a separate commit makes more sense to me!

Reply:
Split a separate commit for this before merging.

Reply:
Thanks! I will note that the other test in this commit did fix a failing test with the changes to `narrow_state.stream_sub`. I thought you were asking about this line specifically, which I changed to match the above.

Because `get_filtered_and_sorted_user_ids` returns `sort_users`, which was previously using `narrow_state.stream_sub` to get  the channel ID. But the test was only overriding `narrow_state.stream_id`, so the test previously wasn't doing any sorting of the buddy list users in that final step because both the channel ID and DM IDs were undefined/empty.

When I updated `stream_sub` to use `stream_id`, then the test failed until the current user (me) was properly added to the peer data for the channel.